Update aws-kinesis-firehose-logs-source.md (#2940)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source.md

* Stack name
* Sumo Logic Source endpoint URL provided when you created the AWS Kinesis Firehose for Logs Source.
* S3 path prefix (this is the prefix under which all failed logs will go in the S3 bucket).

> ### Subscribe AWS Kinesis Firehose to CloudWatch Log Groups
> 
> Follow the instructions in the sections below to subscripe the AWS Kinesis Firehose stream that was created in the above steps to CloudWatch Log Groups.

Reviewer: ```suggestion
### Subscribe AWS Kinesis Firehose to CloudWatch Log Groups

Follow the instructions in the sections below to subscribe the AWS Kinesis Firehose stream that was created in the above steps to CloudWatch Log Groups.
```

#### Manually subscribe AWS Kinesis Firehose stream to an existing CloudWatch Log Group

If you only need to collect logs from a few additional CloudWatch Log groups, you can manually subscribe the AWS Kinesis Firehose subscripton to an existing CloudWatch Log Group using the instructions below.

1. Log in to the [AWS Management Console](https://s3.console.aws.amazon.com/).
2. Under **Management Tools**, select CloudWatch, then click **Logs** in the left- hand navigation menu.
3. Select the CloudWatch Log Group that you want to stream to Sumo Logic, click **Actions** > **Subscription Filters** > **Create Kinesis Firehose subscription filter**

> 4. In the Create Kinesis Firehose subscription filter page, go to **Choose Destination** section and select the Current account and then select the Kinesis Firehose delivery stream that was created in the above steps.

Reviewer: ```suggestion
4. In the Create Kinesis Firehose subscription filter page, go to the **Choose Destination** section and select the Current account. Then select the Kinesis Firehose delivery stream that was created in the above steps.
```

> 5. Under **Grant permissions** select the existing role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`.

Reviewer: ```suggestion
5. Under **Grant permissions**, select the existing role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template, then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`.
```

> 5. Under **Grant permissions** select the existing role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`.

Reviewer: ```suggestion
5. Under **Grant permissions**, select the existing role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template, then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`.
```

6. In the **Configure log format and filters** section, select a Log format and enter a Subscription filter pattern (Optional) and Subscription filter name.
    :::info
    - If no subscription filter pattern is provided it will stream all the logs present in the log group.
    - Sometimes log format will be specified in the Sumo Logic app specific collection page, so use that specific format otherwise dashboards may not light up.
    :::
7. (Optional) In the **Test Pattern** section, select the log data to test, then click **Test pattern**. If test results look fine, then click **Start Streaming**.

#### Auto-subscribe other log groups to SumoCWLogsLambda function

> If you want to collect logs from multiple Log Groups, you can use Sumo’s LogGroup Lambda Connector to subscribe additional Log Groups to the AWS Kinesis Firehose. To do so, follow the instructions in [Auto-Subscribe AWS Log Groups to a AWS Kinesis Firehose stream](../collect-from-other-data-sources/autosubscribe-arn-destination.md). When you edit the connector parameters, set the 'DestinationArnType' parameter to Kinesis, `DestinationArnValue` parameter to the AWS Kinesis Firehose stream ARN, and the `ARNRole parameter to the role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`..

Reviewer: ```suggestion
If you want to collect logs from multiple Log Groups, you can use Sumo’s LogGroup Lambda Connector to subscribe additional Log Groups to the AWS Kinesis Firehose. To do so, follow the instructions in [Auto-Subscribe AWS Log Groups to a AWS Kinesis Firehose stream](/docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination). When you edit the connector parameters, set the 'DestinationArnType' parameter to Kinesis, `DestinationArnValue` parameter to the AWS Kinesis Firehose stream ARN, and the `ARNRole parameter to the role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`..
```

> If you want to collect logs from multiple Log Groups, you can use Sumo’s LogGroup Lambda Connector to subscribe additional Log Groups to the AWS Kinesis Firehose. To do so, follow the instructions in [Auto-Subscribe AWS Log Groups to a AWS Kinesis Firehose stream](/docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination). When you edit the connector parameters, set the 'DestinationArnType' parameter to Kinesis, `DestinationArnValue` parameter to the AWS Kinesis Firehose stream ARN, and the `ARNRole parameter to the role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`..

Reviewer: ```suggestion
If you want to collect logs from multiple Log Groups, you can use Sumo’s LogGroup Lambda Connector to subscribe additional Log Groups to the AWS Kinesis Firehose. To do so, follow the instructions in [Auto-Subscribe AWS Log Groups to a AWS Kinesis Firehose stream](/docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination). When you edit the connector parameters, set the `DestinationArnType` parameter to Kinesis, the `DestinationArnValue` parameter to the AWS Kinesis Firehose stream ARN, and the `ARNRole` parameter to the role that was created in the above steps that grants CloudWatch Logs permission to put data into your Kinesis Data Firehose delivery stream. If you used the Cloudformation template, then the role name will contain the cloudformation name followed by `-KinesisLogsRole-`.
```
